auto_round/autoround.py

```diff
@@ -582,6 +582,10 @@ def parse_format_to_list(self, format: str) -> list:
         from auto_round.utils import SUPPORTED_FORMATS
 
         for format_ in formats:
+            if "gguf" in format_:
+                from auto_round.export.export_to_gguf.convert import download_convert_file
+
+                download_convert_file()
```

Comment:
There's no need to download this file every time, especially since it requires the latest GGUF, which may involve reinstalling  gguf. How about adding a check first, if the model has been  already supported, we can skip the download?

Reply:
another issue is the downloading may be failed due to network issue or some other issues